content/applications/inventory_and_mrp/inventory/management/misc/batch_transfers.rst

```diff
@@ -1,124 +1,235 @@
+column shows the available stock to fulfill the order. Lastly, the :guilabel:`Done` column specifies
+the products that have been picked, and are ready for the next step.
+
+.. example::
+   The product, `Desk Pad`, from the same batch as the :ref:`example above
+   <inventory/management/batch-transfers-example>`, is only visible in the :guilabel:`Operations`
+   tab because there are no :guilabel:`Reserved` quantities in-stock to fulfill the batch picking.
+
+   Click the :guilabel:`Check Availability` button to search the stock again for available products.
+
+   .. image:: batch_transfers/operations-tab.png
+      :align: center
+      :alt: Show unavailable reserved quantities in the *Operations* tab.
+
+Create backorder
+----------------
+
+On the batch transfer form, if the :guilabel:`Done` quantity of the product is *less* than the
+:guilabel:`Reserved` quantity, a pop-up window appears.
+
+This pop-up window provides the option: :guilabel:`Create Backorder?`.
+
+Clicking the :guilabel:`Create Backorder` button automatically creates a new batch transfer,
+containing the remaining products.
+
+Click :guilabel:`No Backorder` to finish the picking *without* creating another batch picking.
+
+Click :guilabel:`Discard` to cancel the validation, and return to the batch transfer form.
+
+.. image:: batch_transfers/create-backorder.png
+   :align: center
+   :alt: Show the *Create Backorder* pop-up.
+
+.. _inventory/management/barcode_picking:
+
+Process batch transfer: Barcode app
+===================================
+
+Created batch transfers are also listed in the :menuselection:`Barcode` app, accessible by selecting
+the :guilabel:`Batch Transfers` button.
+
+By default, confirmed batch pickings appear on the :guilabel:`Batch Transfers` page. On that page,
+click on the desired batch transfer to open the detailed list of products for the picking.
+
+.. image:: batch_transfers/barcode-batch-transfers.png
+   :align: center
+   :alt: Show list of to-do batch transfers in *Barcode* app.
+
+For the chosen batch transfer, follow the instructions at the top of the page in the black
+background. Begin by scanning the product's barcode to record a single product for picking. To
+record multiple quantities, click the :guilabel:`✏️ (pencil)` icon, and enter the required
+quantities for the picking.
+
+.. note::
+   Products from the same order are labeled with the same color on the left. Completed pickings are
+   highlighted in green.
+
+.. example::
+   In a batch transfer for 2 `Cabinet with Doors`, 3 `Acoustic Bloc Screens`, and 4 `Four Person
+   Desks`, the `3/3` and `4/4` :guilabel:`Units` indicate that the last two product pickings are
+   complete.
+
+   `1/2` units of the `Cabinet with Doors` has already been picked, and after scanning the product
+   barcode for the second cabinet, Odoo prompts the user to `Scan a serial number` to record
+   the unique serial number for :ref:`product tracking <inventory/serial_numbers/configure>`.
```

Comment:
```suggestion
```suggestion
   barcode for the second cabinet, Odoo prompts the user to `Scan a serial number` to record the
   unique serial number for :ref:`product tracking <inventory/serial_numbers/configure>`.
```